MidCamp/midcamp
# Drupal 9

# Description

- Effectively removes `"spress/spress"`, a dependency of [MidCamp/Hatter](https://github.com/MidCamp/Hatter) incompatible with Drupal 9's TWIG version
- Upgrades Redirect module for D9 support
- Adds `drupal/core-recommended` package
- Updates Drupal Core to version 9.3
- Updates all config for Drupal Core's new config key ordering
- Updates Drush
- Updates Drupal Extension
- Removes Drupal Console

# Test instructions

1. Install updated dependencies with `lando composer install`
2. Perform database updates via `lando drush updb -y`
3. Browse the site and confirm it still works
